beets/library.py

```diff
@@ -729,7 +729,8 @@ def remove(self, delete=False, with_album=True):
 
         self._db._memotable = {}
 
-    def move(self, copy=False, link=False, basedir=None, with_album=True):
+    def move(self, copy=False, link=False, basedir=None, with_album=True,
+             fields_to_store=None):
```

Comment:
I'm not sure `move` needs the new parameter—it's a little unintuitive for clients, and the only reason we store here is to update the paths for files.